packages/angular-cli/models/config.ts

```diff
@@ -39,15 +39,15 @@ export class CliConfig extends CliConfigBase<ConfigInterface> {
 
 
   static fromProject(): CliConfig {
```

Comment:
This gives you the version the project was generated with, not the version it is currently using. You can only know that from the installed package.